script/github-release

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env ruby
```

Comment:
For instance, something that hub doesn't let me do right now is upload the compiled asset even if it exists. My script detects the duplicate and deletes the old one under the same filename. This is so botched binaries might get updated.